Explorer/Assets/DCL/Character/CharacterPreview/CharacterPreviewController.cs

```diff
@@ -57,9 +59,9 @@ public void Dispose()
             cameraController.Dispose();
         }
 
-        public void UpdateAvatar(CharacterPreviewAvatarModel avatarModel)
+        public UniTask UpdateAvatar(CharacterPreviewAvatarModel avatarModel, CancellationToken ct)
```

Comment:
Async suffix is missing, this is going to make a test go red :D

Reply:
not really, this is just returning an UniTask